cli/src/main.rs

```diff
@@ -100,13 +103,14 @@ fn parse_args() -> StrResult<Command> {
     }
 
     let help = args.contains(["-h", "--help"]);
+    let font_paths = args.values_from_str("--font-path").unwrap();
```

Comment:
Can this unwrap be triggered by malformed arguments?

Reply:
Yes, the reasonable way should be

```rust
let font_paths = args.values_from_os_str("--font-path", |p| {
    Ok::<_, Infallible>(PathBuf::from(p.to_owned()))
}).unwrap();
```

but it needs manual annotation the `Infallible` type of result, and can't remove the `unwrap` which won't panic forever.

If you prefer this, I can make a change.

Reply:
No, it's fine as is.